### ISSUE TITLE
Setup Review

### ISSUE THREAD

**Issue by TheOtterlord** [fogbender/b2b-saaskit#113]
I followed the setup process, skipping the OpenAI & Stripe setup. Love the process. I hit a couple of bumps, some of which were mine, and others were fixed already.

- The git (ssh) clone failed, so I redid with https. SSH might requires auth even for public repos? I've never used it before, so I've not yet run into this.
- ~~Nit: The propelauth example project name is invalid (`pwf-dev` cannot contain `-`) (same in step 9)~~ already fixed!
- The setup page marks step 6 as complete before I even do anything (noticed past step 3)
- Doesn't explain the flow of creating a posthog project. It could probably use a nod to follow the on-screen instructions, and which type of app to create (I assumed web)
- ~~For step 9, you need to do propelauth before the supabase migration~~ also looks to be fixed!

**Comment by JLarky** [comment]
> The setup page marks step 6 as complete before I even do anything (noticed past step 3)

yeah, that one is tricky one :) we can't really check that from that page since it's just a setting you change on propel website. We were thinking of adding some kind manual "mark as done" for that one
